### PR TITLE
Add inbound provisioning with scim2 tutorial

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/learn/configuring-inbound-provisioning.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Configuring Inbound Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. 
+
+Inbound provisioning provisions users or groups into the WSO2 Identity Server by an external application. These external applications are referred to as service providers. WSO2 Identity Server supports the SCIM API and SOAP-based Web service API standards for inbound provisioning.
```
Comment (gomathyk) [inline en/docs/learn/configuring-inbound-provisioning.md]: into WSO2 Identity Server

File: en/docs/learn/configuring-inbound-provisioning.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    {"emails":[{"type":"home","value":"kim.jackson@gmail.com","primary":true},{"type":"work","value":"kim_j@wso2.com"}],"meta":{"created":"2018-08-15T14:55:23Z","location":"https://localhost:9443/scim2/Users/c8c821ba-1200-495e-a775-79b260e717bd","lastModified":"2018-08-15T14:55:23Z","resourceType":"User"},"schemas":["urn:ietf:params:scim:schemas:core:2.0:User","urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"],"name":{"familyName":"jackson","givenName":"kim"},"id":"c8c821ba-1200-495e-a775-79b260e717bd","userName":"kim"}
+    ```
+
+4. View provisioned users using the user ID. Obtain the user ID from the response shown above. 
+
+    ```
+    curl -v -k --user admin:admin 'https://localhost:9443/scim2/Users/<user-id>'
+    ```
+
+5. Update a provisioned user's user details. The update profile details are returned with the response.
```
Comment (gomathyk) [inline en/docs/learn/configuring-inbound-provisioning.md]: Update a provisioned user's profile details. These updated profile details are now returned in the response.

File: en/docs/learn/provisioning-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. Identity provisioning is key for Identity Federation. Identity federation is a mechanism that allows authentication across different enterprises in different trust domains based on a trust factor. This makes access easy, as users do not have to remember a different set of credentials for every application they use. The following identity provisioning concepts are used in WSO2 Identity Server.
```
Comment (gomathyk) [inline en/docs/learn/provisioning-tutorials.md]: maintaining, and deleting

File: en/docs/learn/configuring-inbound-provisioning.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Configuring Inbound Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles.
```
Comment (gomathyk) [inline en/docs/learn/configuring-inbound-provisioning.md]: maintaining, and deleting

File: en/docs/learn/configuring-inbound-provisioning.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-inbound-provisioning.md]: Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065

File: en/docs/learn/configuring-inbound-provisioning.md (hunk 2 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-inbound-provisioning.md]: Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065

File: en/docs/learn/provisioning-tutorials.md (hunk 3 above)
Comment (sherenewso2) [reply en/docs/learn/provisioning-tutorials.md]: Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065

File: en/docs/learn/configuring-inbound-provisioning.md (hunk 4 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-inbound-provisioning.md]: Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065

**Pull request merged**